siliconcompiler/core.py

```diff
@@ -421,16 +422,25 @@ def create_cmdline(self,
             if not switchlist or any(switch in switchlist for switch in switchstrs):
                 used_switches.update(switchstrs)
                 if typestr == 'bool':
-                    parser.add_argument(*switchstrs,
-                                        nargs='?',
-                                        metavar=metavar,
-                                        dest=dest,
-                                        const='true',
-                                        help=helpstr,
-                                        default=argparse.SUPPRESS)
-                # list type arguments
-                elif re.match(r'\[', typestr):
-                    # all the rest
+                    # Boolean type arguments
+                    if pernodestr == 'never':
```

Comment:
Is this definitely the value for setting `-pernode` to false?

Reply:
yes